tutorials/notebooks/astropy-coordinates/1-Coordinates-Intro.ipynb

```diff
@@ -0,0 +1,634 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Astronomical Coordinates 1: Getting Started with astropy.coordinates\n",
+    "\n",
+    "## Authors\n",
+    "Adrian Price-Whelan\n",
+    "\n",
+    "## Learning Goals\n",
+    "* Create `astropy.coordinates.SkyCoord` objects using coordinate data and object names\n",
+    "* Use SkyCoord objects to become familiar with object oriented programming (OOP)\n",
+    "* Use a `SkyCoord` object to query the *Gaia* archive using `astroquery`\n",
+    "* Output coordinate data in different string representations\n",
+    "* Demonstrate working with 3D sky coordinates (including distance information for objects)\n",
+    "\n",
+    "## Keywords\n",
+    "coordinates, OOP, astroquery, gaia\n",
+    "\n",
+    "\n",
+    "## Summary\n",
+    "\n",
+    "Astronomers use a wide variety of coordinate systems and formats to represent sky coordinates of celestial objects. For example, you may have seen terms like \"right ascension\" and \"declination\" or \"galactic latitude and longitude,\" and you may have seen angular coordinate components represented as \"0h39m15.9s,\" \"00:39:15.9,\" or 9.81625º. `astropy.coordinates` provides tools for representing the coordinates of objects and transforming them between different systems. \n",
+    "\n",
+    "In this tutorial, we will explore how the `astropy.coordinates` package can be used to work with astronomical coordinates. You may find it helpful to keep [the Astropy documentation for the coordinates package](http://docs.astropy.org/en/stable/coordinates/index.html) open alongside this tutorial for reference or additional reading. In the text below, you may also see some links that look like ([docs](http://docs.astropy.org/en/stable/coordinates/index.html)). These links will take you to parts of the documentation that are directly relevant to the cells from which they link. \n",
+    "\n",
+    "*Note: This is the 1st tutorial in a series of tutorials about astropy.coordinates.*\n",
```

Comment:
Maybe need another blank line here. I see this:

<img width="1017" alt="Astronomical_Coordinates_1__Getting_Started_with_astropy_coordinates_—_astropy-tutorials_v3_0_dev" src="https://user-images.githubusercontent.com/1147167/100893793-5d8ed780-3481-11eb-8fdc-6e34a4a7f6f8.png">

Reply:
Bug in the rendered version...hopefully will be resolved with the site redesign! Here is what it looks like in a live notebook:
![image](https://user-images.githubusercontent.com/583379/116294365-e32f5e80-a765-11eb-9600-ef23c2aea95a.png)